docs/operators/jsonFormatter.md

```diff
@@ -0,0 +1,25 @@
+Json Formatter
+=============
+
+## Operator Objective
+Purpose of this operator is to consume Plain Old Java Object ("POJO") and write them as JSON.
+This operator is based on [jackson-databind](https://github.com/FasterXML/jackson-databind) and so users can make use of [annotations](https://github.com/FasterXML/jackson-annotations).
+Json Formatter does not hold any state and is **idempotent**, **fault-tolerance** & **statically/dynamically partitionable**.
```

Comment:
omit "does not hold any state" since the claim is incorrect: the base class has state.

Also, describe how the user can enable both kinds of partitioning (or provide links if that is already described elsewhere).